juriscraper/pacer/email.py

```diff
@@ -169,19 +170,20 @@ def _parse_docket_number(
         if self._is_appellate():
             path = self._sibling_path("Case Number")
             case_number = self._xpath_text_0(current_node, f"{path}/a")
+            self.raw_docket_numbers.add(case_number)
```

Comment:
I think this is not required since`raw_docket_numbers` are not being used for parsing appellate `short_descriptions`?